docs/sidebars.js

```diff
@@ -57,6 +57,7 @@ const sidebars = {
         'advanced/logging',
         'advanced/custom-routes',
         'advanced/receiver',
+        'advanced/app-assistant'
```

Comment:
I don't think this needs to be under advanced: it's a basic feature and one that works pretty easily out of the box, as opposed to the other ones under this section.